cmd/dartboard/main.go

```diff
@@ -52,9 +52,22 @@ func main() {
 				Action:      subcommands.Deploy,
 				Flags: []cli.Flag{
 					&cli.BoolFlag{
-						Name:  subcommands.ArgSkipApply,
-						Value: false,
-						Usage: "skip `tofu apply`, assume apply was already called",
+						Name:        subcommands.ArgSkipApply,
+						Value:       false,
+						Usage:       "skip 'tofu apply', assume apply was already called",
```

Comment:
(why) do we need this? tofu apply is expected to be idempotent, so running it twice should be fast and harmless ™️